jwst/cube_build/cube_build.py

```diff
@@ -16,7 +16,6 @@ class CubeData():
 
     def __init__(self,
                  input_models,
-                 input_filenames,
```

Comment:
This change in the API is, I believe, responsible for all the unit tests that are throwing hard errors. The unit tests need updating. See the CI test results for the errors.